package.json

```diff
@@ -30,6 +30,7 @@
     "debug": "^4.1.1",
     "resolve": "^1.15.0",
     "simple-dom": "^1.4.0",
+    "simple-html-tokenizer": "^0.5.9",
```

Comment:
FWIW, I _think_ this is probably blocked on landing https://github.com/tildeio/simple-html-tokenizer/pull/71

Reply:
I only picked simple-html-tokenizer because that's what the bikeshed landed on in the previous PR. 

IMO it would be better to use a more fully-featured off-the-shelf HTML parser and avoid issues like this. It only runs once at server boot, it's not like it's on any performance critical path.

Reply:
Seems fine, but we either need to land that PR or use a different parser. As it stands, we cannot parse properly formatted HTML.